LEAF_Request_Portal/templates/reports/LEAF_Inbox.tpl

```diff
@@ -299,7 +299,7 @@
             // index by roles
             for(let depID in dataInboxes[sites[i].url][j].unfilledDependencyData) {
                 let uDD = dataInboxes[sites[i].url][j].unfilledDependencyData[depID];
-                let roleID = depID;
+                let roleID = depID + dataInboxes[sites[i].url][j].stepID;
```

Comment:
This concerns me.  If both ids are ints, you could get a sum back.  And when I first scanned this, it's what I expected.  But pretty sure it's supposed to be a concat.  So, would probably be better as depID.concat('',  dataInboxes[sites[i].url][j].stepID)    Just need to make sure both are strings.  This is something that shouldn't be a problem now if at least one is a string, but could open up issues later on.  At the very least, it reduces confusion down the road.

Reply:
Nice catch. Updated to make sure it's a string.